org.deidentifier.arx.appdata.xml

```diff
@@ -23,12 +23,14 @@ ARX is able to handle large datasets on commodity hardware and it features an in
   </description>
   <screenshots>
     <screenshot type="default">
-      <caption>jExifToolGUI on GNU/Linux</caption>
-      <image type="source">https://arx.deidentifier.org/wp-content/uploads/2015/12/front_page_screen_2.png</image>
+      <caption>ARX on GNU/Linux</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.deidentifier.arx/master/arx-on-linux.png</image>
```

Comment:
yeah, this has not been merged yet.
Also, this is not a stable URL which might make building in the future harder.
Using https://raw.githubusercontent.com/flathub/org.deidentifier.arx/b416ecb7e9b4a21d7c9af8dd6064bd9a0b8a8d0a/arx-on-linux.png should work better for both reasons

Reply:
Thanks. Done.